### DISCUSSION TITLE
mDNS address

### DISCUSSION THREAD

**Discussion by tknx** [blakeblackshear/frigate#3303] [Ideas]
Would be nice for my SO to have an easy to remember .local address readily available.

**Comment by NickM-27** [comment]
I think this is something that users can achieve. https://gitlab.com/viraptor/docker_mdns

I may be wrong but I believe frigate would need to be run as a host container to do that from within the container and may have other complications

**Reply by tknx** [reply]
Yeah the developer comment on that one was "home, shit code" which made me not want to go there.

**Reply by NickM-27** [reply]
Lol yeah I meant it more as a proof of concept. I did find https://github.com/Jack12816/docker-mdns which may be handleable on the frigate side, just not sure on what other changes would be required.
